osquery/tables/system/linux/deb.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Let's rename this file, as well as it's corresponding implementation, to something more descriptive than `deb`. Perhaps `deb_package_helpers`? Maybe `deb_helpers`?